src/components/TwoLineEntry.js

```diff
@@ -13,18 +13,19 @@ export default class TwoLineEntry extends React.Component {
     message: PropTypes.element,
     onClick: PropTypes.func,
     panel: PropTypes.element,
-    highlight: PropTypes.bool
+    highlight: PropTypes.bool,
+    isDisabled: PropTypes.bool
   }
 
   static defaultProps = {
     onClick: () => {}
   }
 
   render() {
-    const { buttons, image, headline, message, onClick, letter, panel, highlight } = this.props
+    const { buttons, image, headline, message, onClick, letter, panel, highlight, isDisabled } = this.props
     return (
       <div className="two-line-entry">
-        <div className="list-row" onClick={onClick}>
+        <div className={`list-row${isDisabled ? ' isDisabled' : ''}`} onClick={onClick}>
```

Comment:
suggest we change from "disabled" to "empty" or some such. We might actually want the user to be able to do some things on empty entries (e.g., in theory they could start adding curations)